_posts/2017-02-27-rgsoc.md

```diff
@@ -0,0 +1,53 @@
+categories:
+---
+Servo is taking part in the [Rails Girls Summer of Code](http://railsgirlssummerofcode.org/) program as a
+mentoring project once again. We're really excited about the program after our wonderful experience last year! 
+
+## What is Rails Girls Summer of Code?
+
+The Rails Girls Summer of Code program (RGSoC) takes place each year between July and September. Developers form pairs and
+apply to be matched with a mentoring open source project - if a team is selected for a project, they will contribute to
+that project for those three months while receiving a stipend and regular coaching in return.
+
+## How is this program different from similar Summer of Code programs?
+
+Unlike [Google Summer of Code](https://developers.google.com/open-source/gsoc/), one of RGSoC's explicit goals is to increase
+the diversity of the open source community. This means that there are
+[some restrictions](https://railsgirlssummerofcode.org/students/application/#eligibility) on who is eligible to participate.
+Additionally, RGSoC is not limited to university and college students. This provides opportunities to people who are
+transitioning into software development roles and could benefit from several months of focused, real-world experience to
+improve their skills. Finally, RGSoC makes participants pair up and collaborate with coaches to provide the best possible
+environment for making meaningful contributions.
+
+## What does it mean that Servo is participating?
+
+Servo has been accepted as a [mentoring project](https://teams.railsgirlssummerofcode.org/projects/161-servo), which means
+that we want to work with a team to help them implement missing features, such as
+[responsive images](https://github.com/servo/servo/issues/11416) and the
+[OffscreenCanvas](https://github.com/servo/servo/wiki/Offscreen-canvas-project) API. We have only three requirements:
+
+* you are comfortable using a programming language (eg. any of Python, JavaScript, Java, Ruby, C, etc.)
+* you are enthusiastic about learning [Rust](http://www.rust-lang.org/) (which will be used for all changes to Servo)
+* you are unafraid of learning enough HTML/CSS/JavaScript to interact with our [automated tests](https://github.com/servo/servo/tree/master/tests/wpt/web-platform-tests)
+
+If you fit that profile and are [eligible](https://railsgirlssummerofcode.org/students/application/#eligibility) to participate,
+then we would love to see an application with your name on it! The deadline for applying to the RGSoC program is March 8, so
+[don't delay](https://railsgirlssummerofcode.org/students/application/#components)!
+
+## What is it like contributing to Servo in this program?
+
+Last year, [Jeena Lee](http://jeenalee.com/) and [Malisa Smith](http://hellomalisa.me/) implemented
+the [Fetch API](https://fetch.spec.whatwg.org/) as part of RGSoC. You can read about their experience in a
+[talk](http://jeenalee.com/2016/10/26/donutjs.html) they gave about it; you can also read their
```

Comment:
> You can _read_ about their experience in a _talk_ they gave...
This sounds a bit funny (reading about a talk, even though the actual link leads to a blogpost).

Maybe putting it something like this might be better:
> You can learn more about their (journey|experiences) from the talk and blogpost they have presented [...]